### PR TITLE
Disable damage in chunks with shooting disable

### PR DESCRIPTION
**Pull request by OcZi**
See #9 for more information

Tested and working in freeplay lobby. Main menu lobby is not affected

### CODE REVIEW

**Comment by OcZi** [comment]
Reverting Test command because it would not be used for testing

**Review by desertkun — CHANGES_REQUESTED** [review]
Could you please rebase your branch and look into comments?

File: core/src/com/desertkun/brainout/data/components/HealthComponentData.java [hunk 1]
```diff
@@ -65,6 +67,13 @@ public void damage(DamageEvent e)
             return;
         }
 
+        Map map = getMap();
```
[the remark this answers is not in the document]
Comment (OcZi) [reply core/src/com/desertkun/brainout/data/components/HealthComponentData.java:70]: Done. shootingFlag condition will execute now if map is not null

File: core/src/com/desertkun/brainout/data/components/HealthComponentData.java [hunk 2]
```diff
@@ -65,6 +67,13 @@ public void damage(DamageEvent e)
...
+        ChunkData chunk = map.getChunkAt((int) e.x, (int) e.y);
+        if (chunk.hasFlag(ChunkData.ChunkFlag.shootingDisabled)) {
+            // Log.info("Damage blocked by chunk shooting-disable flag");
```
[the remark this answers is not in the document]
Comment (OcZi) [reply core/src/com/desertkun/brainout/data/components/HealthComponentData.java]: Done

**Review by OcZi — COMMENTED** [review]
Everything should be ok if we only do the check when the map exists

**Review by desertkun — APPROVED** [review]

**Comment by desertkun** [comment]
@OcZi FYI the project has not an extensive CI that prepares an isolated environment with a build for every pull request. Just select Client build from the CI, download the artifact, and then double-click on the jar

**Pull request merged**